src/sentry/models/files/abstractfile.py

```diff
@@ -219,7 +219,8 @@ class Meta:
 
     def _get_chunked_blob(self, mode=None, prefetch=False, prefetch_to=None, delete=True):
         return ChunkedFileBlobIndexWrapper(
-            self.FILE_BLOB_INDEX_MODEL.objects.filter(file=self)
+            # TODO: file blob inheritance hierarchy is unsound
```

Comment:
talked a bit with @markstory about these particular models -- we believe there's a way to convert these to free functions to capture the dependent-types inheritance hierarchy with the type system and make these sound -- but as a followup